faq/databases-for-redis.mdx

```diff
@@ -0,0 +1,75 @@
+  paragraph: Discover Scaleway Managed Databases and how to access it for Redis™.
+image: database-redis.webp
+---
+
+## What is Scaleway Databases for Redis™
+
+Databases for Redis™ is a low-latency caching solution. It allows you to easily set up a secure cache and lighten the load on your main database. Based on the in-memory data storage, Databases for Redis™ improves your application response time and helps you provide a better experience to your users. 
+The resource optimizes the speed of your requests as copies of the most frequently used data are stored in memory making them accessible in milliseconds. Learn more about Redis™ databases.
+
+## Why is it ideal for Cache usage?
+
+Based on the **Re**mote **Di**ctionary **S**erver technology, Scaleway Databases for Redis™ stores your data in the RAM of the underlying machine rather than on a disk (SSD/HDD). In other words, for each request to read, to insert, or to update data in a database, this can be executed using data available in the fastest and closest storage of your compute resource, the memory.
+Traditional databases like MySQL or PostgreSQL store data on a disk which inevitably introduces IOPs and results in latency on each operation. Redis™ is known for delivering millisecond response time and high performance for millions of requests per second to empower demanding workloads. 
+The combination of a powerful in-memory data storage such as Redis™ and the management of the resources set-up, securisation, scaling and maintenance makes Scaleway Databases for Redis™ a handy solution to improve the usability of your application 
+One of the most common ways to implement cache is storing frequently accessed data in Redis™ (therefore in memory), and serving your application's request from there, should data be available, instead of from your traditional database. If data is not available in memory, it can always be retrieved from the primary database. Below is a simplified schema explaining this process . 
+
+## How can I access my Redis™ once it is provisioned?
+
+You can connect to your Redis™  using the IP address.es and port.s available in the details page of your Redis™ Instance next to the title **Endpoints**.
+- There are as many `IP:port` as nodes in your cluster (one if standalone).
+- If you are in cluster mode (3 nodes or more), you have to use a cluster specific connector cable to connect to multiple endpoints.
+
+ADD SCREENSHOT.
+
+## What is the Private Networks feature for Redis™ Instances?
+
+[Private Networks](https://www.scaleway.com/en/docs/network/vpc/quickstart/) allows you to configure your own Layer-2 network with IPv4 addresses on Databases for Redis™ Instances. 
+You can therefore enhance the security of your system architecture by isolating it from the Internet. It also increases the security of your databases, since Instances in your Private Network can directly be connected to your Redis™ Instance, without passing through the public Internet.
```

Comment:
```suggestion
It also increases the security of your databases, since Instances in your Private Network can directly be connected to your Redis™ Instance, without passing through the public Internet.
```